pandas/tests/scalar/timestamp/test_unary_ops.py

```diff
@@ -424,3 +424,14 @@ def test_timestamp(self):
             # should agree with datetime.timestamp method
             dt = ts.to_pydatetime()
             assert dt.timestamp() == ts.timestamp()
+
+
+@pytest.mark.parametrize("fold", [0, 1])
+def test_replace_preserves_fold(fold):
+    # GH 37610. Check that replace preserves Timestamp fold property
+    tz = gettz("Europe/Moscow")
+
+    ts = Timestamp(year=2009, month=10, day=25, hour=2, minute=30, fold=fold, tz=tz)
```

Comment:
Why `tz=` instead of `tzinfo=`? I seem to remember that there are vague plans to remove `tz=` as redundant or something?

Reply:
tz kwarg allows for a string whereas tzinfo requires a tzinfo object

Reply:
But that's not what's happening here...

Reply:
@jbrockmendel @pganssle 
Agreed, passing a tzinfo object into the tz argument gives the same result, but passing tzinfo is more readable and futureproof.
Changed it, please take a look.